src/components/Form/TextInput.tsx

```diff
@@ -42,17 +42,17 @@ export interface TextInputProps extends Omit<HtmlInputProps, 'type'> {
   /** Label */
   labelText: string;
   /** Label displaymode
-   * default: show above, screenreader: show as placeholder
+   * visible: show above, hidden: show as placeholder
    */
```

Comment:
Maybe add `@default 'visible'` to JSDoc as it seems to be default when not telling the `labelMode`